syncopy/tests/test_continuousdata.py

```diff
@@ -77,6 +77,109 @@
     + list(zip(["foilim"] * len(foilimSelections), foilimSelections))
 
 
+# Local helper function for performing basic arithmetic tests
+def _base_op_tests(dummy, ymmud, dummy2, ymmud2, dummyC, operation):
```

Comment:
As far as I get this, you somehow try to abstract these selection tests to be  at least operation agnostic?! Would be great to have sth like this for general frontend tests, as this looks suuper cumbersome and error prone to re-implement each time

Reply:
Yes, agreed. This is a long-standing request for our test, cf. #12 (closed this recently since I still don't know how to make this work tbh)